spec/src/main/asciidoc/interoperability.asciidoc

```diff
@@ -61,11 +61,8 @@ upn:: This MP-JWT custom claim is the user principal name in the java.security.P
     principal name in javax.security.enterprise.identitystore.IdentityStore. If this claim is missing, fallback to
     the http://openid.net/specs/openid-connect-core-1_0.html#StandardClaims["preferred_username", OIDC Section 5.1] should be
     attempted, and if that claim is missing, fallback to the "sub" claim should be used.
-groups:: This MP-JWT custom claim is the list of group names that have been assigned to the principal of the MP-JWT.
-    This typically will required a mapping at the application container level to application deployment roles, but a
-    one-to-one between group names and application role names is required to be performed in addition to any
-    other mapping.
 
+[NOTE] It is recommended that JWT tokens contain a `groups` claim if the endpoint requires Authorization. Tokens without this claim can be accepted if the endpoint requires Authentication only. Support for mapping the groups from other claims is optional.
```

Comment:
Shouldn't we introduce the mapper SPI or the configuration that the groups can be taken from another claim here also.

Reply:
@rdebusscher #94 was opened awhile back, but I've marked it now for JWT 2.0. I'd like to release `1.2` asap so that we can go ahead with breaking things and try to make it for MP 4.0 with MP JWT 2.0. This is why I'm postponing everything that can be postponed including the builder API, even though it does not break anything. The only other minor improvement I'd like to try to squeeze is the ES256 support in order to ensure that when the builder api arrives at some point it will be synced (as per your earlier comments) with the server side support.
So, it was a long way to suggest to address the mapper spi post 1.2 :-)